cal/cal/static/js/app.js

```diff
@@ -326,11 +337,7 @@ function CategoryListCtrl($scope, $http, CalendarFilterService, CategoryService)
 
   this.hideZeroHoursFilter = function(value, index, array) {
     /* jshint unused:vars */
-    if (this.hideZeroHours && value.hours === 0) {
-      return false;
-    } else {
-      return true;
-    }
+    return !this.hideZeroHours || value.hours !== 0
```

Comment:
I think there was a reason why we keep it as an expanded if else. The one liner is a bit hard to understand. In English, not hours or hours not equal 0 is pretty confusing...